Linphone/Presentation.Dto/AgentsCommandServiceChangeStatusResponse.cs

```diff
@@ -17,5 +17,5 @@
 
 namespace BelledonneCommunications.Linphone.Presentation.Dto
 {
-    public class CallsCommandServiceTerminateIncomingResponse : GenericResponseBase<CallViewModel> { }
+    public class AgentsCommandServiceChangeStatusResponse : Response { }
```

Comment:
Why does not inherit from `GenericResponseBase` ?

Reply:
I will leave a Todo comment on this, we should work on this in CallCenter panel also.
GenericResponseBase needs a Viewmodel, which wasn't our concern at this point.

Reply:
This issue will be fixes in #9 (God Willing)